app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CustomerItemsController < ApplicationController
+  def create
+    @customer = Customer.find(params[:id])
+    if params[:item_id] != nil
```

Comment:
nice work with this error handling. Just so you know, we don't expect you to do much error handling during assessments but it's great practice to do so if you have time! Great job